docs/guides/kubernetes-dashboard.md

```diff
@@ -10,363 +10,104 @@ description: >-
-
-```sh
-$kubectl apply -f docs/recipes/yml/dashboard-forwardauth.ingress.yaml
-```
-
-<<< @/examples/yml/dashboard-forwardauth.ingress.yaml
-
-```sh
-$kubectl apply -f docs/recipes/yml/dashboard-proxied.ingress.yaml
-```
-
-<<< @/examples/yml/dashboard-proxied.ingress.yaml
-
-And finally, check that the ingresses are up and running.
-
-```sh
- $kubectl get ingress
-```
-
-```sh
-NAME                    HOSTS                                                ADDRESS   PORTS     AGE
-dashboard-forwardauth   dashboard-forwardauth.domain.example                               80, 443   42h
-dashboard-proxied       dashboard-proxied.domain.example                               80, 443   42h
-helm-pomerium           *.domain.example,authenticate.domain.example                   80, 443   42h
-```
-
-You'll notice this is the step where we put everything together. We've got [nginx] handling the initial requests, [cert-manager] handling our public certificates, and Pomerium handling access control.
+    ```bash
+    helm upgrade --install pomerium pomerium/pomerium --values pomerium-values.yaml
+    ```
 
 ## Conclusion
 
-Though the net result will be similar between using forward-auth and direct proxying, there are a few differences:
-
-- By having Pomerium **directly proxy the requests**, you as an administrator have control control over the underlying request. In this example, we are able to inject an authenticating bearer token header to the downstream request which arguably makes for a better user experience.
-
-  <video controls  muted="" playsinline="" width="100%" height="600" control=""><source src="./img/k8s-proxied-example.mp4" type="video/mp4">
-  Your browser does not support the video tag.
-  </video>
-
-* Conversely, going the **forward-auth** route potentially means using the ingress / reverse proxy you are are already accustomed to or have already modified to support your particular deployment.
-
-  <video controls  muted="" playsinline="" width="100%" height="600" control=""><source src="./img/k8s-fwd-auth-example.mp4" type="video/mp4">
-  Your browser does not support the video tag.
-  </video>
+Because we've defined RBAC for our users, they can authenticate with Pomerium and Kubernetes will recognize that user in the Dashboard:
 
-In the end, you should choose whichever option makes the most sense for your use-case and environment.
+<video controls  muted="" playsinline="" width="100%" height="600" control=""><source src="./img/k8s-dashboard-user.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
 
-Whichever option you choose to go with, 🎉🍾🎊 **congratulations** 🎉🍾🎊! You now have a single-sign-on enabled [Kubernetes Dashboard] protected by Pomerium and automatically renewing [LetsEncrypt] certificates.
+🎉🍾🎊 **Congratulations** 🎉🍾🎊! You now have a single-sign-on enabled [Kubernetes Dashboard] protected by Pomerium.
```

Comment:
```suggestion
🎉🍾🎊 **Congratulations!** 🎉🍾🎊 You now have a single-sign-on enabled [Kubernetes Dashboard] protected by Pomerium.
```
- Propose moving the ! mark into the bold so it's closer. The triple emojis seem to be a good enough spacer anyways.

Reply:
IMHO I would get rid of all the emoji and whatnot, it has no place in documentation... But it was there when I got here and I don't want to be the killjoy.